docs/docs/learn/shards/coding-with-shards.md

```diff
@@ -2,6 +2,121 @@
 
 In this chapter, we will be learning how to code with Shards so that you can write your very own program!
 
+## The shard
+
+A shard in its most basic code form consists of its name surrounded by brackets.
```

Comment:
I find "brackets" to be too generic. Depending on the audience (native UK English speaker vs native US speaker vs rest of the world), it might mean something different by default (see https://en.wikipedia.org/wiki/Bracket). For instance, for me the default bracket is the square one as I always use "parentheses" for the "round brackets".

So either use the word parenthesis (pl. parentheses) or qualify them as "round brackets".